test/run_fuse_test.go

```diff
@@ -59,15 +59,21 @@ func createUserFuse(tb testing.TB, ith int, config *libkbfs.ConfigLocal,
 	}
 	tb.Logf("FUSE HasInvalidate=%v", mnt.Conn.Protocol().HasInvalidate())
 
-	ctx := libkbfs.BackgroundContextWithCancellationDelayer()
+	ctx, cancelFn := context.WithCancel(context.Background())
```

Comment:
Is this canceled anywhere? Looks like it could leak...

Reply:
@jzila Yep, it's canceled here: https://github.com/keybase/kbfs/blob/f5d4ec2036bb405200148d293a06d5fc74e7adea/test/engine_fs_test.go#L57

(Via the test shutdown.)

Reply:
Thanks!